src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/web/VersionPageTests.kt

```diff
@@ -90,7 +91,8 @@ class VersionPageTests : IntegrationTest()
 
             it.dsl.select(REPORT_VERSION.REPORT, REPORT_VERSION.ID)
                     .fromJoinPath(REPORT_VERSION_DATA, REPORT_VERSION, FILE_INPUT)
-                    .where(REPORT_VERSION.PUBLISHED.eq(true))
+                    .joinPath(REPORT_VERSION, ORDERLYWEB_REPORT_VERSION)
```

Comment:
Given the widespread effect of this and the join, is it worth making a view at all?